roles/openshift_daemonset_config/templates/daemonset.yml.j2

```diff
@@ -0,0 +1,123 @@
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: ops
+  annotations:
+    kubernetes.io/description: |
+      This daemon set manages the operational configuration for a cluster and ensures all nodes have
+      a concrete set of config in place. It could also use a local ansible run against the /host directory.
+spec:
+  selector:
+    matchLabels:
+      confighosts: ops
+      openshift.io/role: ops
```

Comment:
Probably should be using `ops.openshift.io/role: operations`.  We reserve openshift.io for dev stuff, and it's very possible we may use that in the future.

Generally I recommend always using `app: NAMEOFDAEMONSET` in this case as your label, unless you have a really concrete reason not to.

Reply:
Sounds good, will update.